src/ert/analysis/_es_update.py

```diff
@@ -43,6 +46,99 @@ class SmootherSnapshot:
     update_step_snapshots: Dict[str, "UpdateSnapshot"] = field(default_factory=dict)
 
 
+def _get_A_matrix(
+    temporary_storage: Dict[str, "npt.NDArray[Any]"],
```

Comment:
What else can the array contain except floats, that is, should it be "npt.NDArray[np.float_]"?

Reply:
should be np.double, but yes, it was just me trying to get mypy to work.